redash/settings/helpers.py

```diff
@@ -34,3 +35,13 @@ def int_or_none(value):
         return value
 
     return int(value)
+
+
+def fix_redis_url(url):
```

Comment:
I should rename this to be more descriptive (`add_decode_responses_option`) and make sure it doesn't add the option twice (?).

Reply:
https://github.com/getredash/redash/pull/4232/commits/1ca1268b13efe714b1be42f6aa57902024a1ac59